src/radio/README.md

```diff
@@ -0,0 +1,193 @@
+# RadioGroup Component
+
```

Comment:
can we add a one-liner on the purpose of the component?

Reply:
`A radio control that is used to select a single option from set.`